JapaneseFilter/sections/general_extensions.txt

```diff
@@ -303,6 +303,8 @@ nicovideo.jp#$#.BottomContainer.InView { min-height:auto !important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/34774
 blog.esuteru.com#$#.siteContent { margin-top: 0!important; }
 blog.esuteru.com#$#.siteRanking-main { margin-left: 0!important; }
+! https://github.com/AdguardTeam/AdguardFilters/pull/131231
+otakomu.jp#$#.siteRanking-main { margin-left: 0!important; }
```

Comment:
You can merge with `blog.esuteru.com#$#.siteRanking-main { margin-left: 0!important; }` and replace the comment `! https://github.com/AdguardTeam/AdguardFilters/issues/34774` with `! https://github.com/AdguardTeam/AdguardFilters/pull/131231`, they are essentially the same site. `blog.esuteru.com#$#.siteContent { margin-top: 0!important; }` and `otakomu.jp#$#.jackFix + div.siteContent { margin-top: 0 !important; }` can also be merged as `blog.esuteru.com,otakomu.jp#$#.jackFix + div.siteContent { margin-top: 0 !important; }`

Reply:
All the rest are fine, good work!

Reply:
will merge the rules after merging PR

Reply:
thank you